pkg/controller/instance/instance_controller.go

```diff
@@ -137,7 +136,7 @@ func eventFilter() predicate.Funcs {
 }
 
 func isForPipePod(e event.DeleteEvent) bool {
-	return e.Meta.GetAnnotations() != nil && funk.Contains(e.Meta.GetAnnotations(), task.PipePodAnnotation)
```

Comment:
there are a number of changes that are not tied to 1.18 it appears

Reply:
The branch name is badly named in this case. I set out to update KUDO so that it would fix the deprecation issue with the Kubernetes API that prevented upgrading to Kubernetes 1.22+. That change itself required an update to the k8s libraries which caused a cascading effect of other dependencies that needed to be updated.

So yes it's much bigger than just a GO 1.18 update. I couldn't find an easy way to isolate just the deprecation fix.